### PR TITLE
bump common version to v2

### PR DESCRIPTION
**Pull request by calvin0327**
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/clusterpedia-io/clusterpedia-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/clusterpedia-io/clusterpedia-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/clusterpedia-io/clusterpedia-helm/blob/main/CONTRIBUTING.md).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by calvin0327** [comment]
There is test results:

Postgresql:

```shell
helm install clusterpedia -n test --create-namespace ./charts/clusterpedia/ --set persistenceMatchNode=None --set postgresql.primary.persistence.enabled=false --debug
```
image

Mysql:

```shell
helm install clusterpedia -n test --create-namespace ./charts/clusterpedia/ --set persistenceMatchNode=None --set mysql.primary.persistence.enabled=false --debug
```

image

**Comment by calvin0327** [comment]
@Iceber PTAL

File: charts/clusterpedia/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.9.1
+version: 1.9.2
```
Comment (Iceber) [inline charts/clusterpedia/Chart.yaml]: Looks like a major version change for common, we may need to upgrade to 2.0

**Comment by Iceber** [comment]
Why upgrade to v2.0?
common is a core dependency of clusterpedia and affects the user's values settings, so to ensure that it doesn't cause incompatible changes for users, and to ensure that users can fix some of the issues on v1
ref pr: https://github.com/bitnami/charts/pull/11830

**Review by Iceber — APPROVED** [review]
LGTM, Thanks

**Review by 27149chen — APPROVED** [review]

**Comment by 27149chen** [comment]
LGTM, Thanks

**Pull request merged**